Gamma_Analysis.py

```diff
@@ -0,0 +1,339 @@
+"""
+This will perform a gamma analysis
+It will collect two spectra:
+M = Measurement Spectrum
+B = Background Spectrum
+"""
+from __future__ import print_function
+from SpectrumFileBase import SpectrumFileBase
+import SPEFile
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import os
+# exec('Isotope_identification.py')
```

Comment:
Don't use `exec`, it's unclear and confusing because you don't know what it will put into the namespace.

The better way to do this is just `import Isotope_identification as ii` or so, and you can access the class as `ii.Isotope` and the defined isotopes as `ii.Potassium_40`, etc.